### PR TITLE
Coverity fixes

### LINKED ISSUE #10328: CID 1518875: Unchecked return value
```
 76  // Associate our config file with remap.config to be able to initiate reloads
 77  TSMgmtString result;
 78  const char *var_name = "proxy.config.url_remap.filename";
   	
CID 1518875 (#1 of 1): Unchecked return value (CHECKED_RETURN)
7. check_return: Calling TSMgmtStringGet without checking return value (as is done elsewhere 4 out of 5 times).
 79  TSMgmtStringGet(var_name, &result);
 80  TSMgmtConfigFileAdd(result, configloc.c_str());
```

### LINKED ISSUE #10356: CID 1518576: Logically dead code
NextHopConsistentHash.cc
```
342    case NH_EXHAUST_RING:
343    default:
   	const: At condition wrapped, the value of wrapped must be equal to 0.
   	dead_error_condition: The condition !wrapped must be true.
344      if (!wrapped) {
345        cur_ring = result.last_group;
   	
CID 1518576 (#1 of 1): Logically dead code (DEADCODE)
dead_error_line: Execution cannot reach this statement: if (this->groups > 1U) {
  ....
346      } else if (groups > 1) {
347        cur_ring = (result.last_group + 1) % groups;
348      }
349      break;
350    }
```

### PR DESCRIPTION
**Pull request by ezelkow1**
Check return codes on TS funcs in mmdb.cc
fixes #10328

Remove dead condition in nexthop
fixes #10356

### CODE REVIEW

**Comment by ezelkow1** [comment]
[approve ci osx]

File: plugins/experimental/maxmind_acl/mmdb.cc [hunk 1]
```diff
@@ -78,11 +78,16 @@ Acl::init(char const *filename)
...
   const char *var_name = "proxy.config.url_remap.filename";
-  TSMgmtStringGet(var_name, &result);
-  TSMgmtConfigFileAdd(result, configloc.c_str());
+  if (TS_SUCCESS == TSMgmtStringGet(var_name, &result)) {
+    if (TS_SUCCESS != TSMgmtConfigFileAdd(result, configloc.c_str())) {
+      TSWarning("[%s] Error adding mgmt config file", PLUGIN_NAME);
+    }
+  } else {
+    TSWarning("[%s] Could not retrieve remap filename", PLUGIN_NAME);
+  }
```
Comment (bneradt) [inline plugins/experimental/maxmind_acl/mmdb.cc:90]: This is super nit picky, but we can keep the boolean checks consistent and avoid some nesting like so:

```cpp
 if (TS_SUCCESS != TSMgmtStringGet(var_name, &result)) {
    TSWarning("[%s] Could not retrieve remap filename", PLUGIN_NAME);
  } else if (TS_SUCCESS != TSMgmtConfigFileAdd(result, configloc.c_str())) {
    TSWarning("[%s] Error adding mgmt config file", PLUGIN_NAME);
  }
```

I'll leave it up to you. I'll approve as is if you want to go forward with what you have.

**Review by bneradt — APPROVED** [review]
Looks good.

File: plugins/experimental/maxmind_acl/mmdb.cc (hunk 1 above)
Comment (ezelkow1) [reply plugins/experimental/maxmind_acl/mmdb.cc:90]: oh sure thats fine, let me change it

**Review by bneradt — APPROVED** [review]

**Pull request merged**